Makefile

```diff
@@ -0,0 +1,165 @@
+PROJECT_NAME=fabric8-env-client
+CUR_DIR=$(shell pwd)
+TMP_PATH=$(CUR_DIR)/tmp
+INSTALL_PREFIX=$(CUR_DIR)/bin
+VENDOR_DIR=vendor
+ifeq ($(OS),Windows_NT)
+include ./.make/Makefile.win
+else
+include ./.make/Makefile.lnx
+endif
+SOURCE_DIR ?= .
+SOURCES := $(shell find $(SOURCE_DIR) -path $(SOURCE_DIR)/vendor -prune -o -name '*.go' -print)
+
+# Find all required tools:
+GIT_BIN := $(shell command -v $(GIT_BIN_NAME) 2> /dev/null)
+DEP_BIN_NAME := dep
+DEP_BIN_DIR := ./tmp/bin
+DEP_BIN := $(DEP_BIN_DIR)/$(DEP_BIN_NAME)
+DEP_VERSION=v0.4.1
```

Comment:
I know we are using `v0.4.1`, but how about updating dep to use `v0.5.0` here and in Dockerfile as well. It's just matter of using latest.

Reply:
done